build/common.project.props

```diff
@@ -143,7 +143,6 @@
 
   <PropertyGroup Condition=" '$(Version)' == '' ">
     <Version>$(SemanticVersion)$(PreReleaseInformationVersion)</Version>
-    <Version Condition="'$(BUILD_SOURCEVERSION)' != '' AND '$(IsCommandLinePackage)' != 'true'">$(SemanticVersion)$(PreReleaseInformationVersion)+$(BUILD_SOURCEVERSION)</Version>
```

Comment:
In case anyone in my team sees this and wonders why, there's no benefit to adding the commit hash to the end of the string. Looking at nuget.org, no other packages do this, including System.* packages from the dotnet team. Our nuspec still has the commit hash, to enable source link, so anyone who really wants the commit hash can still get it.